designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/theme/Theme.kt

```diff
@@ -137,19 +137,21 @@ fun DHIS2Theme(content: @Composable () -> Unit) {
 
 internal object DHIS2SCustomTextStyles {
 
-    val titleBold = TextStyle(
+    val titleMediumBold = TextStyle(
```

Comment:
I see you've modified the variable but I don't see the variable renamed in the usage, can you make sure it is modified too if it is being used? same with the one below (bodyLargeBold)

Reply:
it wasn't in use